Installation/debian/arangodb.init

```diff
@@ -33,6 +33,10 @@ start () {
     ( cd /var/lib/arangodb3 && chown -R arangodb:arangodb . && chmod 700 . ) || exit 1
     ( cd /var/lib/arangodb3-apps && chown -R arangodb:arangodb . && chmod 700 . ) || exit 1
     ( cd $PIDDIR && chown arangodb:arangodb . && chmod 755 . ) || exit 1
+    ( cd /var/lib/arangodb3-apps && chown -R arangodb:arangodb . && chmod 700 . ) || exit 1
```

Comment:
why does it set the permissions for `arangodb3-apps` a second time? it does already in line 34 if I am not wrong.